manifests/user.pp

```diff
@@ -16,8 +16,9 @@
   $wrapper      = '/usr/bin/zpr_wrapper.py',
 ) inherits zpr::params {
 
-  # For placement of keys manually
+  include zpr::rsync_cmd
 
+  $permitted_commands = "${home}/.ssh/permitted_commands"
```

Comment:
Where does the .ssh/permitted_commands file come from? I've looked through the repo and can't find a reference.